services/bookstore/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build the manager binary
+FROM golang:1.14.1 as builder
+
+ARG workDirName=/github.com/aws/aws-service-operator-k8s
+WORKDIR $workDirName
+# For building Go Module required
+ENV GOPROXY=direct
```

Comment:
"Each ENV line creates a new intermediate layer" 

You can merge all ENVs in a single line to optimize the build.

Reply:
Yes, it was a design in older docker. But in latest best-practices docker it ADD, RUN and COPY creates intermediate layer. That and this way i think it is more readable readable too. Let me know if you have strong opinion about it. thanks

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
Nice!